src/components/SearchTracePage/TraceSearchForm.js

```diff
@@ -106,7 +106,7 @@ export function submitForm(fields, searchTraces) {
   let start;
   let end;
   if (lookback !== 'custom') {
-    const unit = lookback[1];
+    const unit = lookback.substr(-1);
```

Comment:
seems like we need a unit test

Reply:
added double-digit parse [test](https://github.com/jaegertracing/jaeger-ui/pull/168/commits/5db4d6a251aec6880831a1ec5b57ac23f73f5a0c#diff-7478a808394cdea4c7237c63c8cc13b4R165)